content/en/docs/Reference/veneers.md

```diff
@@ -0,0 +1,397 @@
+---
+title: "Veneers"
+linkTitle: "Veneers"
+weight: 5
+date: 2022-06-30
+---
+
+File-Based Catalogs (FBC) are a major improvement to the imperative update graph approaches of previous versions. FBCs give operator authors a [declarative and deterministic approach to defining their update graph](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/). However, FBCs can get complex, especially as the number of releases and dependencies scale.
+
```

Comment:
This guide is an in-depth walkthrough to let you know how to work with veneers. Veeners are APIs which will help Operator authors and Index catalog admins generate and keep maintained their content using the [FBC][] format.

@grokspawn IHMO we need to try to be more direct here. Who is the audience? Why they will look at this doc? We need to make the info clear and less verbose as possible for who use it and try to abstract complexity that only matters to those keep it maintained.

Example: `Enter the `veneer`, a simpler transitional step to generating an FBC.` 

What means a transitional step to generate an FBC for those that are just looking to publish their solutions in OperatorHub io or OCP catalogs?